backend/columnar.js

```diff
@@ -1736,7 +1736,7 @@ class BackendDoc {
       valLen = 9, valRaw = 10, predNum = 13, predActor = 14, predCtr = 15, succNum = 13, succActor = 14, succCtr = 15
 
     const objectId = ops.objId
-    const elemId = op[keyStr] ? op[keyStr] :
+    const elemId = op[keyStr] ? op[keyStr] : // eslint-disable-line
```

Comment:
There is a rule (which I disabled) which makes it an error to nest ternary operators because they're difficult to read & reason about. I think we probably ought to enable that, which would make the actual rule (something about the colon being in an inconsistent place) irrelevant here.